src/bin.ts

```diff
@@ -13,21 +18,18 @@ function getArg(name: string) {
 
 // For getting a token
 const creds = getArg('creds');
+const org = getArg('org');
 const owner = getArg('owner');
 const repo = getArg('repo');
 
 // For generating a creds bundle
 const cert = getArg('cert');
 const appId = getArg('app-id');
 
-if (creds && owner && repo) {
-  getTokenForRepo(
-    {
-      owner,
-      name: repo,
-    },
-    appCredentialsFromString(creds),
-  )
+if (creds && (org || (owner && repo))) {
```

Comment:
We should probably ensure these are exclusive.  i.e. `--org` should not allow `--owner` 🤔 

Or maybe just switch to `--org` and `--repo` and make `--repo` optional while still supporting `--owner` as a fallback for `--org` 🤔

Reply:
Good catch. I went with the first option because I'm both lazy, and I think we should leave `--owner` since it can be used to generate tokens for non-org repos (i.e. personal ones) so the usage of `--org` there would be a bit confusing.